guides/source/action_mailer_basics.md

```diff
@@ -533,6 +533,24 @@ url helper.
 NOTE: non-`GET` links require [jQuery UJS](https://github.com/rails/jquery-ujs)
 and won't work in mailer templates. They will result in normal `GET` requests.
 
+### Adding images in Action Mailer Views
+
+Unlike controllers, the mailer instance doesn't have any context about the
+incoming request so you'll need to provide the `:asset_host` parameter yourself.
+
+As the `:asset_host` usually is consistent across the application you can
+configure it globally in config/application.rb:
+
+```ruby
+config.action_mailer.asset_host = 'http://example.com'
+```
+
+Now you can display a image inside you email as usual. For example
```

Comment:
* s/a image/an image
* s/you email/your email
* Remove 'as usual' and 'For example'